docs/cisco_node_utils.yaml.example

```diff
@@ -0,0 +1,28 @@
+# The entry labeled 'default' will be used unless another entry is specified
+# default:
+#   host: foo
```

Comment:
Just to be clear:  If I have a node `default` and three nodes `foo1`, `foo2`, `foo3`, will the command `rake test` still run the minitests against the `default` node?